apps/docs/app/examples/defaultValues.tsx

```diff
@@ -9,12 +9,14 @@ import { SubmitButton } from "~/components/SubmitButton";
 export const validator = withZod(
   z.object({
     firstName: z
-      .string()
-      .nonempty("First name is required"),
-    lastName: z.string().nonempty("Last name is required"),
+      .string({ required_error: "First name is required" })
```

Comment:
Unless the semantics of required changed as well, these will all have to be:

```ts
z.string().min(1, { message: "First name is required" })
```

This is because the value in the `FormData` will always be at least an empty string, so a true `required_error` will never actually be seen.